src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -198,13 +198,42 @@ async function stopMinikube(context: Context): Promise<void> {
     });
 }
 
+function getStatusFormatString(currentVersion: string): string {
+    if (currentVersion.startsWith("v0.")) {
+        if (parseInt(currentVersion.split(".")[1]) < 31) {
+            // minikube < 0.31.0 uses this format
+            return `"{{.MinikubeStatus}}","{{.ClusterStatus}}","{{.KubeconfigStatus}}"`;
+        } else {
+            // minikube < 1.0.0 uses this format
+            return `"{{.Host}}",{ "kubelet": "{{.Kubelet}}", "apiserver": "{{.ApiServer}}" },"{{.Kubeconfig}}"`;
+        }
+    }
+
+    // minikube >= 1.0.0 uses this format
+    return `"{{.Host}}",{ "kubelet": "{{.Kubelet}}", "apiserver": "{{.APIServer}}" },"{{.Kubeconfig}}"`;
```

Comment:
Later on we determine `MinikubeInfo.isRunning` by comparing the first element of the JSON array to `Stopped`.  What does `{{.Host}}` return?  Is the comparison to `Stopped` meaningful in this case?

Reply:
see https://github.com/Azure/vscode-kubernetes-tools/pull/575#discussion_r300956555; btw i did not change the logic here, so i'm not sure if it makes sense.

anyhow, as far as i understand minikube:
host: Stopped/Running -> status of the vm
kubelet: Stopped/Running -> status of the kubelet in the minikube vm
apiserver: Stopped/Running -> status of the apiserver of k8s in the minikube vm

and for the old return values i'd guess that "clusterstatus" is "kubelet" and "apiserver" combined. i'd have to verify this in the minikube docu/code but sounds right to me.

Reply:
Sounds good - I wasn't sure what `Host` contained but you have confirmed it's a status field so that suffices for me!  As you say if we want to get fancier with the logic that's outside the scope of this PR.